src/ListOfContacts/Dto/ListOfContactListsHubSpotEntity.cs

```diff
@@ -0,0 +1,71 @@
+﻿using Newtonsoft.Json;
+using Skarp.HubSpotClient.Core.Interfaces;
+using System;
+using System.Collections.Generic;
+using System.Runtime.Serialization;
+using System.Text;
+
+namespace Skarp.HubSpotClient.ListOfContacts.Dto
+{
+    public class ListOfContactListsHubSpotEntity: IHubSpotEntity
+    {
+        [DataContract]
+        public class MetaData
+        {
+            public string processing { get; set; }
+            public int size { get; set; }
+            public string error { get; set; }
+            public DateTime lastProcessingStateChangeAt { get; set; }
+            public DateTime lastSizeChangeAt { get; set; }
+        }
+
+        [DataContract]
+        public class List

```

Comment:
**suggestion** I would rename this - it is a very generic name and it clashes with a CLR type. Also from what I can see it is used as a generic argument for an _actual list_ - perhaps a better name could be `ListItem` or `LisftOfListsItem` to avoid too generic names.